server/backend/mlx.py

```
@@ -329,197 +94,328 @@ async def generate_response_chat_stream(
+        "text": {"format": {"type": "text"}, "verbosity": "low"},
+        "paralell_tool_calls": 0,
+        "truncation": "disabled",
+        "tool_choice": "auto",
+        "error": error,
+    }
+    created_response.update(_get_commons_responses(request))
+    return created_response
+
+
+def _get_commons_responses(request: ResponsesRequest):
+    if request.tools != None:
```

Ruff (0.15.14)

[error] Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)